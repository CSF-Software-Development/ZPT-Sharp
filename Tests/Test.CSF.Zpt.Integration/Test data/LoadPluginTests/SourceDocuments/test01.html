<html>
<head>
<title>Load plugin test 1</title>
</head>
<body>
<div tal:define="doc load:here/Documents/shared01.html"
     tal:content="structure doc">
  <p>
    This content should all be replaced by the loaded template.
  </p>
</div>
</body>
</html>