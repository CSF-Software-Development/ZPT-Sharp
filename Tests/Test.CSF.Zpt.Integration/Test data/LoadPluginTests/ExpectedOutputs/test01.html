<html>
<head>
<title>Load plugin test 1</title>
</head>
<body>
<div><html>
<head>
<title>Shared doc 1</title>
</head>
<body>
<p>
  Hello Joseph Bloggs!

</p>
</body>
</html></div>
</body>
</html>